src/main/java/org/ibp/api/rest/labelprinting/SubObservationDatasetLabelPrinting.java

```diff
@@ -54,12 +55,10 @@
 import java.util.Objects;
 import java.util.Optional;
 import java.util.Set;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
 
 @Component
 @Transactional
-public class SubObservationDatasetLabelPrinting extends LabelPrintingStrategy {
+public class SubObservationDatasetLabelPrinting extends ObservationDatasetLabelPrinting {
```

Comment:
@cuenyad is this for the common static fields between both? anything different between them?

Reply:
@clarysabel 
Both have more or less the same labels, so the methods to get the values are the same. Is the same that happens with GermplasmLabelPrinting  and GermplasmListLabelPrinting